app/views/merchant/items/index.html.erb

```diff
@@ -1,10 +1,10 @@
-<h1><%= @items[0].merchant.name %> Items</h1>
+<h1><%= @merchant.name %> Items</h1>
 <section class="grid-container">
-  <% @items.each do |item| %>
+  <% @merchant.items.each do |item| %>
     <section class = "grid-item" id= 'item-<%=item.id%>'>
       <h2><%=link_to item.name, "/items/#{item.id}" %></h2>
       <%=link_to image_tag(item.image), "/items/#{item.id}" %>
-      <p> <%= item.description unless @merchant%></p>
+      <p> <%= item.description%></p>
```

Comment:
refactor here. `unless @merchant` is completely unnecessary here as we are here _as_ a merchant